src/render_engine/vulkan/vulkan_render_engine_init.cpp

```diff
@@ -10,20 +10,18 @@
 #include "vulkan_utils.hpp"
 
 namespace nova::renderer {
-    vulkan_render_engine::vulkan_render_engine(const nova_settings& settings, nova::ttl::task_scheduler* task_scheduler)
-        : render_engine(settings, task_scheduler) {
+    vulkan_render_engine::vulkan_render_engine(const nova_settings& n_settings, nova::ttl::task_scheduler* task_scheduler)
```

Comment:
Don't need to change the name. The compiler is able to disambiguate `settings` from `this->settings` in the constructor's initializer list